waltz-test/src/main/java/com/wepay/waltz/test/util/IntegrationTestHelper.java

```diff
@@ -297,7 +312,19 @@ public WaltzStorageRunner getWaltzStorageRunner(int port, int adminPort, int jet
      * @throws Exception
      */
     public StorageAdminClient getStorageAdminClient() throws Exception {
-        return getStorageAdminClient(storageAdminPort);
+        return getStorageAdminClientWithPort(0);
```

Comment:
guess you want to `return getStorageAdminClient(0)`?

Reply:
Oh thanks! It looks like I messed up refactoring there.